requirements.txt

```diff
@@ -1,6 +1,7 @@
 sphinx>=4.5
 markupsafe
 pydantic>=2.0
+pydantic-core>=2.10
```

Comment:
I think pydantic-core always needs to be exactly in sync with pydantic, so it might make more sense to just specify a constraint on pydantic.

Reply:
I could pin to pydantic v2.4 instead (accomplishes the same version lock)